### DISCUSSION TITLE
autocmd to disable plugin on bufenter

### DISCUSSION THREAD

**Discussion by olaberglund** [LunarVim/LunarVim#2675] [Q&A] [answered]
Hey! I recently installed an additional plugin and am trying to disable it upon opening a markdown file with the help of autocmd. I just tried changing the value to 'false' in a callback function, but it didn't do the trick. Any suggestions?

![nw](https://user-images.githubusercontent.com/64544704/170983768-8232695e-0f56-4489-8a4a-d1fd1692536e.png)

**Comment by LostInTheLogs** [comment, marked as answer]
setting that dynamically has no effect, check the plugin docs for other ways to disable it
